config/rbac_policy.csv

```diff
@@ -0,0 +1,9 @@
+p, user, /api/v1/movies, read
+p, admin, /api/v1/movies, read
+p, admin, /api/v1/movies, write
+p, user, /api/v1/logger, read
+p, admin, /api/v1/logger, read
+p, admin, /api/v1/logger, write
+g, otto.maddox711@gmail.com, admin
+g, mr.nguyenhien1994@gmail.com, admin
```

Comment:
I think we should remove any real emails from this library. Users should add those locally or in their own copies. Let's leave the otto.maddox email and the user_xyz email for examples